src/System.Diagnostics.Process/tests/ProcessTestBase.cs

```diff
@@ -66,6 +66,15 @@ protected Process CreateProcessInfinite()
             });
         }
 
+        protected Process CreateProcessInfinite()
+        {
+            return CreateProcess(() =>
+            {
+                while (true)
+                    Thread.Sleep(WaitInMS);
```

Comment:
No loop is needed here.  If the goal is just to get this to sleep forever (until the process is killed), it could be:
```C#
Thread.CurrentThread.Join();
```
or:
```C#
new ManualResetEventSlim().Wait();
```
or other such things.